kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SettingUpKolibri.vue

```diff
@@ -162,6 +162,18 @@
           settings: omitBy(settings, v => v === null),
           preset: this.wizardContext('formalOrNonformal') || 'nonformal',
           language_id: currentLanguage,
+
+          // // clarify -- does device name char count include "Person Device of" or is that just rendered in the UI?
+
+          // // is there anywhere else device name can get assigned to full name (saw nickname elsewhere)
+
+          // // suggested fix(-ish)
+          // let baseTranslatedLength = this.$tr('defaultDeviceName', { name: '' }).length
+          // let truncatedFullName = this.user.full_name.slice(0, baseTranslatedLength-1);
+          // let actualStringWeUse = this.$tr('defaultDeviceName', { name: truncatedFullName });
+
+          // // if there is a device name available, use it, otherwise get superuser's full name
+          // // and insert into (translation of) 'personal device of {}'
```

Comment:
Looking back at this now I guess we could just truncate what we send in any case below when we're assigning the `device_name` field.

So you could take line 179 here and just cut it down to 50 characters to appease all of the validators. This should resolve the originally reported error where long names break the final step of the setup and hopefully won't be an issue for users as I believe it will only be used when showing the device on a network (ie, during sync / import / remote browsing).

We'll need to apply this to the Facility Name too I believe and should be good.